internal/pkg/aws/ecs/ecs.go

```diff
@@ -168,6 +181,58 @@ func (e *ECS) ServiceTasks(clusterName, serviceName string) ([]*Task, error) {
 	return tasks, nil
 }
 
+// DefaultCluster returns the default clusters in the account and region.
+func (e *ECS) DefaultCluster() (string, error) {
+	resp, err := e.client.DescribeClusters(&ecs.DescribeClustersInput{})
```

Comment:
I think if we want this to actually be the default cluster, we have to include `default` in the list of cluster names passed in.

Otherwise we just get *all* the clusters and we're just grabbing a random one. 
https://docs.aws.amazon.com/sdk-for-go/api/service/ecs/#DescribeClustersInput

Reply:
My understanding from the doc is that if we don't specify a cluster, the default cluster is assumed. I also tested it - if I provide an empty `DescribeClustersInput`, the client will actually return the one default cluster (named "default"), instead of all clusters.

With that being said though, it is indeed easier to follow if we explicitly specify `default` in the input. My concern was that if they change the way in which they identify the default cluster (right now it is simply the cluster named `default`, but what if it is changed to some other names in the future), we would have to make the changes accordingly.

Reply:
Ah! Such a good catch - thank you!!! You're right :)

> If you do not specify a cluster, the default cluster is assumed.